narwhal/config/src/lib.rs

```diff
@@ -519,6 +519,15 @@ impl WorkerCache {
             .collect()
     }
 
+    /// Returns the addresses of all workers
```

Comment:
nit: adjust comment to reflect its "all others workers". Looks like you can remove the [all_workers](https://github.com/MystenLabs/sui/blob/faa08ec3993f818e94563659039189c23c96093f/narwhal/config/src/lib.rs#L495-L504) method above?